AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -143,6 +143,9 @@ private extension SessionHeaderView {
                     }) {
                         Image(systemName: chevronIndicator)
                             .renderingMode(.original)
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: 15, height: 15)
```

Comment:
This one here is mind-blowing, I was trying to accomplish it so hard a long time ago. Big congrats, it's a game changer here 🥳